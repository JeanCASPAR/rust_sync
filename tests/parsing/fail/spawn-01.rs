use rustre::sync;

sync! {
    #![pass(0)]

    node oui(c : bool) = ()
    where
        a : int = spawn 0;
}

fn main() {}
