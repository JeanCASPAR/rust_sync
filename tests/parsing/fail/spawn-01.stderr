error: expected identifier
 --> tests/parsing/fail/spawn-01.rs:8:25
  |
8 |         a : int = spawn 0;
  |                         ^
